client/wfprev-war/src/main/angular/.gitignore

```diff
@@ -44,4 +44,7 @@ Thumbs.db
 documentation.json
 
 # Package Lock
-package-lock.json
\ No newline at end of file
+package-lock.json
```

Comment:
We shouldn't be ignoring the lock files I don't believe.  Please remove both.  These files are what ensures every environment gets the same version of the dependencies.

Reply:
Sounds good, I haved added a messages.ts as shared file to use in diffrent components